docs/README.md

```diff
@@ -0,0 +1,44 @@
+# Why are we building OLM v1?
+
+Operator-lifecycle-manager's mission has been to manage the lifecycle of cluster extensions centrally and declaratively on Kubernetes clusters. Its purpose has always been to make installing, 
```

Comment:
This should be:
```
Operator Lifecycle Manager's mission...
```
Use hyphen's for adjectives, but for a noun/proper name, it should not have hyphens and should be capitalized.

Reply:
This is marked as resolved but is still showing as `Operator-lifecycle-manager's ...` instead of `Operator Lifecycle Manager's...` - was this marked resolved in error?